app/adapters/alma_adapter/bib_item_set.rb

```diff
@@ -13,51 +13,6 @@ def initialize(items:, adapter:)
       @adapter = adapter
     end
 
-    def each(&block)
```

Comment:
A few of these seem like they could be nice to have around, but I guess we can always put them in again if we need them and if they're not tested they're probably a liability. Just curious your thoughts here for `each`, `holding_ids`, `mms_id`.

Reply:
Keeping `each` seems nice, otherwise it's not really a composite object. The others seem fine to remove until we need them.

Reply:
OK. There was no test for it, so I removed it.

Reply:
Added back and tested.